tests/libvmi/factory.go

```diff
@@ -76,11 +75,10 @@ func NewAlpine(opts ...Option) *kvirtv1.VirtualMachineInstance {
 func NewAlpineWithTestTooling(opts ...Option) *kvirtv1.VirtualMachineInstance {
 	// Supplied with no user data, AlpimeWithTestTooling image takes more than 200s to allow login
 	withNonEmptyUserData := WithCloudInitNoCloudEncodedUserData("#!/bin/bash\necho hello\n")
-	alpineMemory := cirrosMemory
 	alpineOpts := []Option{
 		WithContainerImage(cd.ContainerDiskFor(cd.ContainerDiskAlpineTestTooling)),
 		withNonEmptyUserData,
-		WithResourceMemory(alpineMemory()),
+		WithAlpineResourceMemory(),
```

Comment:
thinking out loud: maybe the problem here is that we set `WithCloudInitNoCloudEncodedUserData` by default,
while IMO it should be a fallback in case it is not provided

Reply:
Or define both `NewAlpineWithTestToolingAndUserData` and `NewAlpineWithTestTooling` but I would discourage this.